### ISSUE TITLE
Adding text into textfield Charachter problem

### ISSUE THREAD

**Issue by ErdincBARUT** [ruffle-rs/ruffle#9327]
I trying to add text ("Misafirler kamera açamazlar lütfern üye oldunuz.") into text field with Turkish characters (same text) every adding text  (same text) going to worse.

![charBug](https://user-images.githubusercontent.com/63129026/215261371-79adb68c-a918-4c29-ba2d-4f57411d8910.png)
Version: ActionScript 2 - Flash 8
Ruffle: Web Demo Container.

Console : ruffle_web-wasm_extensions.js:671 WARN core/src/avm1/activation.rs:993 Cannot enumerate Undefined

Thank you.

**Comment by n0samu** [comment]
Can you attach the SWF file that you're trying to use? Please create a zip file containing the SWF file, then upload the zip file here.

**Comment by ErdincBARUT** [comment]
Right Click Mouse -> Play -> Play  ->Play

[newVisual.zip](https://github.com/ruffle-rs/ruffle/files/10527002/newVisual.zip)
![Screen](https://user-images.githubusercontent.com/63129026/215261823-3879e80c-7092-4437-94d1-e36c4a9412d5.png)

**Comment by n0samu** [comment]
Can confirm the issue, thanks for reporting!

**Comment by n0samu** [comment]
I've created a minimal reproduction of the issue:
[TurkishHTMLText.zip](https://github.com/ruffle-rs/ruffle/files/10527114/TurkishHTMLText.zip)
It uses an HTML text box and a button with this AS attached:
```as
on(release) {
	txtBox.htmlText += "Misafirler kamera açamazlar lütfen üye olunuz.";
}
```
This is enough to trigger the same issue.

**Comment by Lord-McSweeney** [comment]
Closed by #9333.
